policyengine_us/variables/gov/states/il/tax/income/il_final_taxation_amount.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class il_final_taxation_amount(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Il Final Taxation Amount"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+    defined_for = StateCode.IL
+
```

Comment:
Nikhil recently added this feature, which we'll be migrating existing variables to. Optional for this PR, but under this syntax we don't need unit tests (you could remove the formula).
```suggestion
    adds = ["il_total_tax"]
    subtracts = ["il_total_payments_and_refundable_credit"]
```